web/viewer.html

```diff
@@ -360,15 +360,28 @@
             </div>
           </div>
         </div>
+<!--#if !(FIREFOX || MOZCENTRAL)-->
+        <div id="printServiceOverlay" class="container hidden">
+          <div class="dialog">
+            <div class="row">
+              <span>Preparing document for printing...</span>
```

Comment:
Good find. I really think we should localize this and I think it's easier now that it's part of `viewer.html`.